components/tidb_query/src/codec/mysql/json/binary.rs

```diff
@@ -1,51 +1,96 @@
 // Copyright 2017 TiKV Project Authors. Licensed under Apache-2.0.
 
+use std::cmp::Ordering;
```

Comment:
Prefer to copy the comment https://github.com/pingcap/tidb/blob/master/types/json/binary.go#L34:L99 to this file to make the maintainer easy to maintain this module.